exonum-java-binding/service-archetype/src/main/resources/archetype-resources/start-testnet.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Starts a test Exonum network with a single validator node.
+#
+# The network files are placed in `$PWD/testnet` directory, which is cleared at the beginning.
+# The Java service artifacts must be placed into 'testnet/artifacts' directory.
+#
+# See https://exonum.com/doc/version/latest/get-started/java-binding/#node-configuration
+# for extra information.
+
+# Prints a section header
+function header() {
+    local title=$1
+    local rest="========================================================================"
+    echo
+    echo "===[ ${title} ]${rest:${#title}}"
+    echo
+}
+
+# Fail immediately in case of errors and/or unset variables
+set -eu -o pipefail
+
+# Check whether the Exonum Java App is installed.
+command -v exonum-java >/dev/null 2>&1 || { echo >&2 "Please install the Exonum Java App and make sure that 'exonum-java' binary is available via PATH. Aborting."; exit 1; }
+
+# Clear test dir.
+TESTNET_PATH="$PWD/testnet"
+rm -rf "$TESTNET_PATH"
+mkdir "$TESTNET_PATH"
+
+# Enable predefined native logging configuration,
+# unless it is already set to any value (incl. null)
+export RUST_LOG="${RUST_LOG-error,exonum=info,exonum-java=info,java_bindings=info}"
+
+header "1 Generate Template Config"
+exonum-java generate-template \
+    --validators-count=1 \
+    --supervisor-mode simple \
+    "$TESTNET_PATH"/common.toml
+
+header "2 Generate Node Private and Public Configs"
+exonum-java generate-config \
+    "$TESTNET_PATH"/common.toml \
+    "$TESTNET_PATH" \
+    --no-password \
+    --peer-address 127.0.0.1:5400
+
+header "3 Finalize Configuration"
+exonum-java finalize \
+    "$TESTNET_PATH"/sec.toml \
+    "$TESTNET_PATH"/node.toml \
+    --public-configs "$TESTNET_PATH"/pub.toml
+
+header "4 Start Test Network"
+exonum-java run \
+    --node-config "$TESTNET_PATH"/node.toml \
+    --artifacts-path "$TESTNET_PATH"/artifacts \
+    --db-path "$TESTNET_PATH"/db \
+    --master-key-pass pass \
+    --public-api-address 127.0.0.1:3000 \
```

Comment:
It makes sense to document/print for user API port numbers

Reply:
Isn't that enough: 
```
[2020-03-02T10:02:21Z INFO  exonum_api::manager] Starting private web api on 127.0.0.1:3010
[2020-03-02T10:02:21Z INFO  exonum_api::manager] Starting public web api on 127.0.0.1:3000
[2020-03-02T10:02:21Z INFO  exonum_api::manager] Server restart requested
```
? 

The thing is, if we print before starting, exonum will produce quite some log output and hide our message, so it won't be that visible (but still, could be found by 
```
===[ START TESTNET ]===========================================================
```
line.

Reply:
Oh, okay, it's pretty good. Though exonum produces a lot of log lines in a short period of time on default block generation speed, so the user can lost these lines.

Reply:
Ok, I almost added this:

```patch
Index: exonum-java-binding/service-archetype/src/main/resources/archetype-resources/start-testnet.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- exonum-java-binding/service-archetype/src/main/resources/archetype-resources/start-testnet.sh	(revision 2db90913df0aeb62e88e08e81b0fcb6d0ec51ab8)
+++ exonum-java-binding/service-archetype/src/main/resources/archetype-resources/start-testnet.sh	(date 1583162884995)
@@ -52,11 +52,18 @@
     --public-configs "$TESTNET_PATH"/pub.toml
 
 header "4 Start Test Network"
+PUBLIC_API_PORT="3000"
+PRIVATE_API_PORT="3010"
+JAVA_API_PORT="7000"
+echo "  Node API"
+echo "  - Public API port: ${PUBLIC_API_PORT}"
+echo "  - Private API port: ${PRIVATE_API_PORT}"
+echo "  - Java API port: ${JAVA_API_PORT}"
 exonum-java run \
     --node-config "$TESTNET_PATH"/node.toml \
     --artifacts-path "$TESTNET_PATH"/artifacts \
     --db-path "$TESTNET_PATH"/db \
     --master-key-pass pass \
-    --public-api-address 127.0.0.1:3000 \
-    --private-api-address 127.0.0.1:3010 \
-    --ejb-port 7000
+    --public-api-address 127.0.0.1:"$PUBLIC_API_PORT" \
+    --private-api-address 127.0.0.1:"$PRIVATE_API_PORT" \
+    --ejb-port "$JAVA_API_PORT"
```

but I think it is redundant because the script is checked in, and the user will read it. I think having a simpler script is better, than having more logs (which they may still miss).